### ISSUE TITLE
Reporting failed method compilations get an ArgumentTypeError

### ISSUE THREAD

**Issue by rjsargent** [GsDevKit/GsDevKit#118]
When reporting the failed method compilations, #_defineClassNamed:with: gets an error because it uses MCDefinition>>#description rather than e.g. #summary. The former returns an array (class name, selector, and a Boolean for whether it is a class method, in the case of a method definition). The latter returns a string such as e.g. Class class>>#_defineClassNamed:with:.

 ```
             failedMethodCompilations
                do: [ :ar |
                  | details def |
                  details := ar at: 1.
                  def := ar at: 2.
                  errorStream
                    tab;
                    nextPutAll: def description , ': ';
                    cr;
                    nextPutAll: details;
                    cr ].
              self error: errorStream contents ] ].
```

See stack frames 10 and 11 in the attached log file.
[ArgumentTypeError.log](https://github.com/GsDevKit/GsDevKit/files/3418922/ArgumentTypeError.log)

**Comment by rjsargent** [comment]
The example shown in the stack report resulted from adding an instance variable to the superclass that was already used as a temporary variable name in the method.

**Comment by dalehenrich** [comment]
merged to master [a52114f]
